mozci/ci_manager.py

```
@@ -138,6 +138,8 @@ def trigger_range(self, buildername, repo_name, revisions, times, dry_run, files
 class TaskClusterManager(BaseCIManager):
 
     def schedule_graph(self, task_graph, *args, **kwargs):
+        if tc.validate_graph(task_graph) is False:
+            raise Exception("The graph provided does not match the schema")
```

Just call validate and let the exception bubble all the way up.